ansible_collections/arista/avd/roles/eos_commands_collect/tasks/main.yml

```diff
@@ -0,0 +1,36 @@
+---
+# tasks file for eos_commands_collect
+
+- name: Create output directory for each EOS device
+  file:
+    path: "{{ commands_backup_dir }}/{{ inventory_hostname }}/fragments"
+    state: directory
+
+- name: run show commands on remote EOS devices
+  eos_command:
+    commands: "{{ item }}"
+  with_items: "{{ commands_to_collect }}"
+  ignore_errors: true
+  register: cli_output
+
+- name: save collected commands in text files
```

Comment:
Would be nice to select which kind of output we want:

- Pure text
- Markdown report
- Json (from what @carlbuchmann did in a demo)

By using a list of expected output formats, it will let user to select multiple choices

Reply:
Added a var to define desired outpit format.  
currently supported format are md report for the fabric, and text files for each command.

Reply:
Need help if we want to add json format report for the fabric

Reply:
Please look at this file as it can be a good source to start a template for your role: https://github.com/arista-netdevops-community/ansible-webinar-february-2020/blob/master/roles/eos_post_fabric_validation/templates/fabric_json_output.j2

Reply:
Done.  Many thanks :-)